cherrypy/_cplogging.py

```diff
@@ -452,6 +452,5 @@ def emit(self, record):
 
 class LazyRfc3339UtcTime(object):
     def __str__(self):
-        """Return now() in RFC3339 UTC Format."""
-        now = datetime.datetime.now()
-        return now.isoformat('T') + 'Z'
+        """Return utcnow() in RFC3339 UTC Format."""
+        return f"{datetime.datetime.utcnow().isoformat('T')}Z"
```

Comment:
Let's keep complex expressions out of f-strings since it increases cognitive load/readability/maintainability.
```suggestion
        iso_formatted_now = datetime.datetime.utcnow().isoformat('T')
        return f'{iso_formatted_now!s}Z'
```